modules/QtPDF/trans/QtPDF_it.ts

```diff
@@ -1087,31 +1088,31 @@
         <location filename="../src/PDFDocumentView.cpp" line="3077"/>
         <source>B</source>
         <extracomment>File size: bytes</extracomment>
-        <translation></translation>
+        <translation>Dimensione del file: byte</translation>
```

Comment:
The actual source text to be translated (if necessary) is given in the `<source>` tag, here "B". The "File size: bytes" is just a comment for translators to try to explain the context of what this "B" means and how it is used. Please translate the `<source>` text, not the comment. This also applies to the following couple of entries.